rewrite-java-tck/src/main/java/org/openrewrite/java/tree/MethodDeclarationTest.java

```diff
@@ -63,6 +63,20 @@ public <P, R> R foo(P p, String s, String... args) {
         );
     }
 
+    @Test
+    void typeVarargs() {
+        rewriteRun(
+          java(
+            """
+              class Test {
+                  void foo(String  ...  args) {
+                  }
+              }
+              """
+          )
+        );
+    }
```

Comment:
`MethodDeclarationTest.typeArguments()` already has a method with varargs signifier. Of course neither that test or this one failed before this fix because the correct result is printed out despite the LST being wrong. So neither test case catches this bug.

Reply:
Right, neither test case catches this bug because the printed results are all correct, so seems the only way to capture it is add some verification in `spec.afterRecipe()`

Reply:
Yes, that should do it.